mlflow/server/js/src/components/MetricView.js

```diff
@@ -63,6 +63,7 @@ class MetricView extends Component {
                             {this.props.runUuids.map((uuid, idx) => (
                                 <Line type="linear"
                                       dataKey={uuid}
+                                      key={uuid}
```

Comment:
This was also causing a React warning that the items in a list did not have distinct keys